tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = 
+#     py36-django{11,20},
+#     flake8,
+    apidocs
+
+skip_missing_interpreters = True
+usedevelop = True
+
+# [testenv]
+# deps =
+#     django11: Django<2.0
+#     django20: Django>=2.0,<2.1
+# 
+# envlist = 
+#     py36-django{11,20}
+# 
+# commands =
+#     python setup.py test
+
+[testenv:flake8]
+deps = flake8
+commands = flake8 --ignore=W391 djongo {posargs}
+
+[testenv:apidocs]
+whitelist_externals =
+    rm
+    cd
+    mkdir
+commands =
+	rm -f apidocs/modules.rst
```

Comment:
the apidocs is living in the root of the project, which I do not want. It should live instead inside the docs/ folder, I have made this change in the main branch.

Reply:
Agreed, I did it like this to prevent clashes and because I wasn't sure if it was going to conflict with the other docs